docs/src/integrations/observability/new-relic/php.md

```diff
@@ -2,11 +2,8 @@
 title: "PHP"
 ---
 
-{{< note title="PHP version">}}
-
-Due to New Relic [dropping support for Zend Thread Safe (ZTS) versions](https://docs.newrelic.com/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9170300/), you can't use New Relic with PHP v8.0+. To use New Relic with PHP, specify `type: 'php:7.4'` in your `.platform.app.yaml` file.
-
-{{< /note >}}
+New Relic isn't yet available on PHP 8.1.
```

Comment:
Do we want to add a link from [the list of extensions](https://pr-2174-in2qtpq-652soceglkw4u.eu-3.platformsh.site/languages/php/extensions.html) to point to this page?

Reply:
If you mean on the extension, given the current styling (links looking the same as not links), I don't think it's worth it.